### PR TITLE
implementation of awesome books using JS classes

### PR DESCRIPTION
**Pull request by ZahraArshia**
On this second part of this awesome books project:
- We used Classes to store all objects and functions.
- the books are generated dynamically with Javascript.
- Any book can be removed with the remove button.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 This the awesome book project built with JavaScript
 
-![](file:///home/moise/awsome-books-project/index.html?)
+![](file:///home/moise/awsome-books-project/index.html)
 
 # Awesome books: plain JavaScript with objects
```
Comment (TedLivist) [inline README.md:10]: - [ ] In your README, kindly define instructions on how to get and run the app locally.

File: index.html [hunk 2]
```diff
@@ -5,21 +5,26 @@
...
+      <input id="author" placeholder="Author" type="text" aria-label="authorName" required/>
+      <br>
+    </form>
     <button class="addButton" id="addButton" type="submit">Add</button>
-  </form>
-
-<script src="./js/script.js"></script>
+  </section>
+  
+<!-- <script src="./js/script.js"></script> -->
```
Comment (TedLivist) [inline index.html]: - [ ] Kindly remove commented code as they make your code look messy and hard to read

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ZahraArshia and @moise-mulungu,

Good job so far!
✔️ Books can be removed and added
✔️ The right book is removed when there are duplicates
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZahraArshia** [comment]
Hello, thanks for the review
required changes implemented.

**Review by rukundoeric — APPROVED** [review]
Hi @ZahraArshia,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**